oeis.py

```diff
@@ -752,6 +752,42 @@ def A002182() -> Iterable[int]:
             yield n
 
 
+@oeis.from_generator(offset=0)
+def A065722() -> Iterable[int]:
+    """Primes that when written in base 4, then reinterpreted in base 10, again give primes."""
+    from sympy.ntheory import prime
+
+    n = 1
+    while True:
+        p = prime(n)
```

Comment:
Couid be rewritten as:

```python
from sympy import sieve

for p in sieve:
```

Reply:
Oh, it would be so much "eat your own dog food"y if we could do:

```
for p in A000040:
```

but it does not work because A000040 starts at 1...

Maybe it could be very easily fixed by adding:

```python
    def __iter__(self):
        for i in count(self.offset):
            yield self[i]
```

in the `IntegerSequence` class, do you want to try this?

Reply:
Alright. So I have switched the implementation to use `sympy.sieve`.

Let me read more about the code and the `IntegerSequence` class.

Reply:
As suggested, I have implemented the `__iter__` dunder method on the `IntegerSequence` class.
Thanks for the heavy lifting work. 🙌